aospy/calc.py

```diff
@@ -865,19 +865,19 @@ def _update_data_out(self, data, dtype):
             self.data_out = {dtype: data}
 
     def save(self, data, dtype_out_time, dtype_out_vert=False,
-             scratch=True, archive=False):
+             save_files=True, save_tar_files=False):
         """Save aospy data to data_out attr and to an external file."""
         self._update_data_out(data, dtype_out_time)
-        if scratch:
-            self._save_to_scratch(data, dtype_out_time)
-        if archive:
-            self._save_to_archive(dtype_out_time)
-        logging.info('\t{}'.format(self.path_scratch[dtype_out_time]))
+        if save_files:
+            self._save_files(data, dtype_out_time)
+        if save_tar_files:
+            self._save_tar_files(dtype_out_time)
+        logging.info('\t{}'.format(self.path_out[dtype_out_time]))
 
     def _load_from_scratch(self, dtype_out_time, dtype_out_vert=False,
```

Comment:
Let's rename this too... `_load_from_disk`?  All "scratch" references should be gone.